source/DEC_API.c

```diff
@@ -21,11 +21,15 @@
 #include "global.h"
 #include "DEC_API.h"
 
+static char dec_last = 0x00;
+
 void DEC_init(){
     LOG("rotencoder init\n");
+    AD1PCFGL       |= 0x1800; /*Set AN11, AN12 to Digital Pins*/ 
```

Comment:
@FlorianHeidecker parameter list updated thx for feedback
re AD1PCFGL better safe than sorry ;-)